content/blog/2017/2017-01-31-pipeline-scalability-best-practice.adoc

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "Best Practices for Scalable Pipeline Code"
+tags:
+- pipeline
+- performance
+- scalability
+author: svanoort
+---
+====
+This by link:https://github.com/svanoort[Sam Van Oort], Software Engineer at
+link:http://cloudbees.com[CloudBees] and contributor to the Jenkins project.
+====
+
+=== Promises
```

Comment:
Remove `=== Promises`. Putting a header tag (`<h3>`) at this point in the blog post is confusing and unecessary